modules/nomad-cluster/main.tf

```diff
@@ -83,6 +83,7 @@ resource "aws_launch_configuration" "launch_configuration" {
     content {
       device_name           = ebs_block_device.value["device_name"]
       volume_size           = ebs_block_device.value["volume_size"]
+      snapshot_id           = ebs_block_device.value["volume_size"] == 0 ? ebs_block_device.value["snapshot_id"] : null
```

Comment:
Should it be checking if `ebs_block_device.value["snapshot_id"]` is `null` instead?

Reply:
Yes, you are right. It is even easier by just taking the snapshot-id from the ebs-config. I have tested it within our infrastructure and it worked with Terraform 0.12.24